content/spin/v2/ai-sentiment-analysis-api-tutorial.md

```diff
@@ -257,27 +255,16 @@ ln -s ~/my-ai-models/ ~/application-one/.spin/ai-models
 
 ### Application Configuration
 
-Then, we configure the `[[component]]` section of our application's manifest (the `spin.toml` file); explicitly naming our model of choice. For example, in the case of the sentiment analysis application, we specify the `llama2-chat` value for our `ai_models` configuration:
-
-```toml
-ai_models = ["llama2-chat"]
-key_value_stores = ["default"]
-```
+Then, we configure the `[component.sentiment-analysis]` section of our application's manifest (the `spin.toml` file); explicitly naming our model of choice. For example, in the case of the sentiment analysis application, we specify the `llama2-chat` value for our `ai_models` configuration, and add a `default` key-value store:
 
-Note the positioning, of the `ai_models` configuration, shown below:
+> Note: `[component.sentiment-analysis]` contains the name of the component. If you used a different name, when creating the application, this sections name would be different.
 
 ```toml
-[[component]]
-id = "sentiment-analysis"
-source = "target/spin-http-js.wasm"
-exclude_files = ["**/node_modules"]
-key_value_stores = ["default"]
+[component.sentiment-analysis]
+...
```

Comment:
Is it intentional that you are no longer showing the surrounding fields?  The previous text said "note the positioning" and I'm not sure what it was aiming at or if that's still relevant.

Reply:
Yes, I didn't see a consistent implementation across all templates, in terms of what was in that section, so decided to so it this way. I don't think it decreases chance of success.